src/v/pandaproxy/schema_registry/service.cc

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+#include "pandaproxy/schema_registry/service.h"
+
+#include "pandaproxy/api/api-doc/schema_registry.json.h"
+#include "pandaproxy/logger.h"
+#include "pandaproxy/schema_registry/configuration.h"
+#include "pandaproxy/schema_registry/handlers.h"
+
+#include <seastar/core/future-util.hh>
+#include <seastar/core/memory.hh>
+#include <seastar/http/api_docs.hh>
+
+namespace pandaproxy::schema_registry {
+
+server::routes_t get_schema_registry_routes() {
+    server::routes_t routes;
+    routes.api = ss::httpd::schema_registry_json::name;
+
+    routes.routes.emplace_back(server::route_t{
+      ss::httpd::schema_registry_json::get_schemas_types, get_schemas_types});
+
+    return routes;
+}
+
+static context_t make_context(
```

Comment:
code comments on this? 

why do we need the *full* core-local memory? 

don't we need at the very list a `//TODO:...` shouldn't we tie this to resources/memory_resources.h 

if the current proxy does this we should create a tix. but configuration should pass in a mem limit.

Reply:
> why do we need the _full_ core-local memory?
> 
> don't we need at the very list a `//TODO:...` shouldn't we tie this to resources/memory_resources.h
> 
> if the current proxy does this we should create a tix. but configuration should pass in a mem limit.

Good catch, I missed that when I integrated Pandaproxy with Redpanda

Reply:
https://github.com/vectorizedio/redpanda/issues/1392

Reply:
registering memory groups should usually be deferred to 30% of the entire core memory. so maybe we limit it to be part of the RPC level memory. just pass that memory group as it exists today no? i'm ok w/ this being in a separate change.

Reply:
a couple of things, can we create a `meta` ticket for schema registry and add #1392 to that. also can we make a comment in code with a // TODO and a reference to 1392